lib/core/src/lib/form/components/widgets/date/date.widget.ts

```diff
@@ -52,33 +55,34 @@ export class DateWidgetComponent extends WidgetComponent implements OnInit, OnDe
 
     DATE_FORMAT = 'DD-MM-YYYY';
 
-    minDate: Moment;
-    maxDate: Moment;
+    minDate: Date;
+    maxDate: Date;
 
     private onDestroy$ = new Subject<boolean>();
 
     constructor(public formService: FormService,
-                private dateAdapter: DateAdapter<Moment>,
-                private userPreferencesService: UserPreferencesService) {
+                private dateAdapter: DateAdapter<DateFnsAdapter>,
```

Comment:
incorrect parameter, should be the value type, not another adapter,
valid cases are `DateAdapter<Moment | Date | string>` etc